_posts/2018-11-30-release-1.7.0.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Apache Flink 1.7.0 Release Announcement"
+date:   2018-11-30 12:00:00
+categories: news
+authors:
+- till:
+  name: "Till Rohrmann"
+  twitter: "stsffap"
+---
+
+The Apache Flink community is pleased to announce Apache Flink 1.7.0. 
+The latest release includes more than 420 resolved issues and some exciting additions to Flink that we describe in the following sections of this post. 
+Please check the [complete changelog](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12343585) for more details.
+
+
+Flink 1.7.0 is API-compatible with previous 1.x.y releases for APIs annotated with the `@Public` annotation.
+The release is available now and we encourage everyone to [download the release](http://flink.apache.org/downloads.html) and check out the updated [documentation](https://ci.apache.org/projects/flink/flink-docs-release-1.7/).
+Feedback through the Flink [mailing lists](http://flink.apache.org/community.html#mailing-lists) or [JIRA](https://issues.apache.org/jira/projects/FLINK/summary) is, as always, very much appreciated!
+
+You can find the binaries on the updated [Downloads page](http://flink.apache.org/downloads.html) on the Flink project site.
+
+{% toc %}
+
+## Flink 1.7.0 - Extending the reach of Stream Processing
+
+In Flink 1.7.0 we come closer to our goals of enabling fast data processing and building data-intensive applications for the Flink community in a seamless way. 
+Our latest release includes some exciting new features and improvements such as support for Scala 2.12, an exactly-once S3 file sink, the integration of complex event processing with streaming SQL and more features that we explain below.
+
+## New Features and Improvements
```

Comment:
I don't know how much we want to go into detail here but we also added an Elasticsearch 6 connector for Table API & SQL/SQL Client. And added a couple of new built in functions.

Reply:
Can you write a short summary about misc table API additions @twalthr? I'll add it then.

Reply:
```
* **Miscellaneous Features for Streaming SQL**:
Besides the major features mentioned above, Flink's Table & SQL API has been extended to serve more use cases.

The following built-in functions were added to the APIs:
`TO_BASE64`, `LOG2`, `LTRIM`, `REPEAT`, `REPLACE`, `COSH`, `SINH`, `TANH`

The SQL Client now supports the [definition of views](https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/table/sqlClient.html#sql-views) both in an environment file and within a CLI session. Furthermore, basic SQL statement auto-completion has been added to the CLI.

The community added an [Elasticsearch 6 table sink](https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/table/connect.html#elasticsearch-connector) which allows to store updating results of a dynamic table.
```

Reply:
@tillrohrmann feel free to edit if you see a need for it.

Reply:
Thanks @twalthr. I'll add it.